locales/fr-FR.json

```diff
@@ -120,6 +120,7 @@
     "title": "Elk est en mode Aperçu !"
   },
   "language": {
+    "none": "Aucun",
```

Comment:
I saw that this translation was missing for all languages and isn't use anywhere

Reply:
Alright, I therefore removed it from all language files.